src/librustc_trans/trans/consts.rs

```diff
@@ -130,13 +132,14 @@ fn addr_of_mut(ccx: &CrateContext,
 
 pub fn addr_of(ccx: &CrateContext,
                cv: ValueRef,
+               align: machine::llalign,
                kind: &str)
                -> ValueRef {
     match ccx.const_globals().borrow().get(&cv) {
         Some(&gv) => return gv,
```

Comment:
A global could be found in the `const_globals` map which has the correct value, but not the correct alignment.